.github/workflows/smoke-test-imperative-compare.yml

```diff
@@ -0,0 +1,124 @@
+name: Imperative Smoke Tests Compare
```

Comment:
Instead of having two CI workflows executing similar tests, can we enrich existing CI tests to add compare functionality and to run the e2e tests?

Reply:
I don’t recall exactly why, but they were temporary created for different purposes for the v6 -> v7 migration.